### PR TITLE
To do structure

### PR DESCRIPTION
**Pull request by elhamatokhi**
## In this pull request I did the following:
-  Used webpack.
- Wrote a function that iterates over an array of objects.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  const displayToDo = todoArr.map(
+    (item) => `
+    <div class="todo item">
+    <input type="checkbox" class="checkbox" />
+    <p>${item.description}</p>
+    <span class="delete" data-id="${item.id}">X</span>
+  </div> `,
+  );
+  lists.innerHTML = displayToDo.join(' ');
+};
```
Comment (danmainah) [inline src/index.js]: - [ ] @elhamatokhi a great job on your app, however when the index is changed the todo items are not displayed according to the index as shown below, kindly implement this feature as its part of the requirement
image
image

File: src/index.html [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+          <h1>Today's To Do...</h1>
+          <div class="btnInput">
+            <input
+              type="text"
+              class="todo-name"
+              id="input-task"
+              data-input
+              placeholder="Add to your list...."
+            />
+            <button class="add-todo">⬅</button>
```
Comment (danmainah) [inline src/index.html]: - [ ] @elhamatokhi great work so far kindly ensure this button icon matches the one given in the design, also the completed button background doesn't match the design given ,below is a comparison:

Design  Given         |  Your design     
:-------------------------:|:-------------------------:
image  | image

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @elhamatokhi , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design


# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/index.js (hunk 1 above)
Comment (elhamatokhi) [reply src/index.js]: Thank you for the kind review. Would you kindly make it a little bit clear? As I did not understand the main requirement. I have traversed the items using their index value.
@danmainah

**Review by sumairq — APPROVED** [review]
### Hi @elhamatokhi , 



Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Good commit history
✔️Linters are passing
✔️Very professional README
✔️Correct Github flow
✔️Descriptive PR
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**